examples/lib/stories/games/games.dart

```diff
@@ -28,5 +29,11 @@ void addGameStories(Dashbook dashbook) {
       (_) => const TRexWidget(),
       codeLink: gamesLink('trex'),
       info: TRexGame.description,
+    )
+    ..add(
+      'Crystal Ball',
```

Comment:
Put it first! It is for sure the coolest

Reply:
Its not an actual game tho